src/cdk/overlay/overlay-directives.ts

```diff
@@ -272,7 +272,12 @@ export class ConnectedOverlayDirective implements OnDestroy, OnChanges {
 
   /** Builds the overlay config based on the directive's inputs */
   private _buildConfig(): OverlayState {
-    let overlayConfig = new OverlayState();
+    const positionStrategy = this._position = this._createPositionStrategy();
+    const overlayConfig = new OverlayState({
```

Comment:
Since all of the options are optional now, could you just change most places to, e.g.,
```ts
const overlayConfig = {
  positionStrategy,
  scrollStrategy: this.scrollStrategy,
  hasBackdrop: this.hasBackdrop,
};
```
?